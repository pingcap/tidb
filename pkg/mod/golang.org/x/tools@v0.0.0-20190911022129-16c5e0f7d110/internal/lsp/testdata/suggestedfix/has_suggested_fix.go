package suggestedfix

import (
	"log"
)

func goodbye() {
	s := "hiiiiiii"
	s = s //@suggestedfix("s = s")
	log.Printf(s)
}
