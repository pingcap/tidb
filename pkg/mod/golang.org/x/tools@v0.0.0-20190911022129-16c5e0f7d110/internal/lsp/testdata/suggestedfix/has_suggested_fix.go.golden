-- suggestedfix --
package suggestedfix

import (
	"log"
)

func goodbye() {
	s := "hiiiiiii"
	 //@suggestedfix("s = s")
	log.Printf(s)
}

